features/step_definitions/protected_area_steps.rb

```diff
@@ -0,0 +1,3 @@
+Given(/^there is a user with username "([^"]*)" and password "([^"]*)"$/) do |usern, passw|
+  User.create(username: usern, password: passw)
```

Comment:
Agree, legacy from yesterday, in case it was a reserved word or something. It is changed now.